07/main07.py

```diff
@@ -0,0 +1,166 @@
+# 1. Створити телефонну книгу, яка матиме наступні команди:
+#
+# stats: кількість записів
+#
+# add: додати запис
+#
+# delete <name>: видалити запис за іменем (ключем)
+#
+# list: список всіх імен в книзі
+#
+# show <name>: детальна інформація по імені
+#
+# Записи не мають повторюватися, заборонено перезаписувати записи, тільки видаляти і додавати заново.
+#
+
+# importing some necessary libs here
+import os
+import time
+# lib import finished
+
+phone_dic = {
+    'entry0': None,
+    'entry1': 10,
+    'entry2': 20,
+    'entry3': 30,
+    'entry4': 40,
+    'entry5': 50,
+}
+
+def dic_count_items(c_var_i):
+    counter = 0
+    for key0,value0 in c_var_i.items():
+        counter += 1
+    return counter
+
+
+# decided to do it in cycle, omg here we go
+while True:
+    os.system('cls')
```

Comment:
What is this for?

Reply:
This clears the screen (like Ctrl+L does). I thought it would be better for user experience.